bertytech/content/blog/2FA/index.md

```diff
@@ -0,0 +1,98 @@
+
+A significant disadvantage of this 2FA is that if the mobile network does not work or is in a region without coverage, you will not receive the code allowing you to identify yourself. 
+Another disadvantage is if you are using connected and centralized equipment (between your tablet and mobile), you will receive the code on these devices; your login codes are as much at risk as your passwords.
+
+
+## 📱 2FA by Authentificator App:
+
+![](https://i.imgur.com/YuRNDY0.jpg)
+
+
+Authentication applications provide you with the same type of service as 2FA by SMS. However, instead of receiving the codes, they are generated locally within your smartphone or tablet. So, they offer flexibility when traveling to a location without cellular service.
+
+The operation relies on cryptographic algorithms to generate single-use passwords that integrate the notion of time (Time-based One-time Password algorithm, TOTP). 
+
+The 2FA application will first generate a start key called "seed". The latter will be derived and mixed with the time and date to make a one-time code, usually valid for 30 to 60 seconds.
+
+> 💡 A good thing to know: there are [password managers](https://berty.tech/blog/best-password-manager/) that manage TOTPs.
+
+***Open Source TOTP:***
+
+👉 https://awesomeopensource.com/projects/totp 
+ 
+👉 https://freeotp.github.io/
+
+
+The major drawback of 2FA applications is that if a scammer grabs your seed (either from your equipment or by hacking the server), he can compute any future code. If you log in and use the authentication application on the same equipment, your authentication codes are as much at risk as your passwords.
+
+
+## 🗝️ FIDO U2F hardware authenticators: YubiKey and others 
+
+![](https://i.imgur.com/ZzJa77a.png)
+
+
+If you are looking for a robust solution that fits your pocket, then look no further than hardware tokens based on the U2F (Universal 2nd Factor) standard, created by the FIDO Alliance. The most famous and typical example of U2F is YubiKey, made by Yubico. 
+
+Concretely, it is a small device the size of a USB key that generates the identification code locally. Just plug in the key to create the right identification code. It is thus an additional step in the real world: the U2F must be physical with you.
+
+However, you will find yourself confronted with a few constraints: you will have to buy it and the system is not supported on all sites that already support 2FA by SMS/Applications. This 2FA requires physical access to the USB port of the machine (so forget the use with remote access like TeamViewer). Finally, if you lose it, you will have to recover all your accounts using the 2FA recovery codes ... Hope you have saved them well. Otherwise, you are good to contact each site's support to have the 2FA disabled on your accounts!
+
+
```

Comment:
you can also speak about websites that remove the password (password-less) and only authenticate you with your email address/phone number) -> tell that it's not 2FA, but in some cases you can prefer having to care about having a secure email provider and system than to have a secure website